code/datums/elements/strippable.dm

```diff
@@ -148,18 +151,20 @@
 	if (isnull(item))
 		return FALSE
 
-	source.visible_message(
-		span_warning("[user] tries to remove [source]'s [item.name]."),
-		span_userdanger("[user] tries to remove your [item.name]."),
-		ignored_mobs = user,
-	)
+	var/is_silent = can_be_silent && HAS_TRAIT(user, TRAIT_STICKY_FINGERS) //SKYRAT EDIT ADDITION
+	if (!is_silent)
+		source.visible_message(
+			span_warning("[user] tries to remove [source]'s [item.name]."),
+			span_userdanger("[user] tries to remove your [item.name]]."),
+			ignored_mobs = user,
+		)
 
 	to_chat(user, span_danger("You try to remove [source]'s [item]..."))
 	user.log_message("[key_name(source)] is being stripped of [item] by [key_name(user)]", LOG_ATTACK, color="red")
 	source.log_message("[key_name(source)] is being stripped of [item] by [key_name(user)]", LOG_VICTIM, color="red", log_globally=FALSE)
 	item.add_fingerprint(src)
 
-	if(ishuman(source))
+	if(ishuman(source) && !is_silent) //SKYRAT EDIT ADDITION original if(ishuman(source))
```

Comment:
Oh, also, for this, did you make sure that it still properly gets logged if they try to take something off of someone that's SSD in the logs? I'd assume so, but you're never too sure.

Reply:
two lines above nerd